tests/test_1490_jax_reducers_combinations.py

```diff
@@ -55,58 +55,59 @@ def func_jax_with_axis(x):
     value_vjp_jax, vjp_func_jax = jax.vjp(func_jax_with_axis, test_regulararray_jax)
 
     numpy.testing.assert_allclose(
-        ak.to_list(value_jvp), value_jvp_jax.tolist(), rtol=1e-9, atol=np.inf
+        ak.to_list(value_jvp), value_jvp_jax.tolist(), rtol=1e-9, atol=1e-9
     )
     numpy.testing.assert_allclose(
-        ak.to_list(value_vjp), value_vjp_jax.tolist(), rtol=1e-9, atol=np.inf
+        ak.to_list(value_vjp), value_vjp_jax.tolist(), rtol=1e-9, atol=1e-9
     )
     numpy.testing.assert_allclose(
-        ak.to_list(jvp_grad), jvp_grad_jax.tolist(), rtol=1e-9, atol=np.inf
+        ak.to_list(jvp_grad), jvp_grad_jax.tolist(), rtol=1e-9, atol=1e-9
     )
     numpy.testing.assert_allclose(
         ak.to_list(vjp_func(value_vjp)[0]),
         (vjp_func_jax(value_vjp_jax)[0]).tolist(),
         rtol=1e-9,
-        atol=np.inf,
-    )
-
-
-@pytest.mark.parametrize("axis", [0, 1])
-@pytest.mark.parametrize("func_ak", [ak.sort])
-def test_sort(func_ak, axis):
-    func_jax = getattr(jax.numpy, func_ak.__name__)
-
-    def func_ak_with_axis(x):
-        return func_ak(x, axis=axis)
-
-    def func_jax_with_axis(x):
-        return func_jax(x, axis=axis)
-
-    value_jvp, jvp_grad = jax.jvp(
-        func_ak_with_axis, (test_regulararray,), (test_regulararray_tangent,)
-    )
-    value_jvp_jax, jvp_grad_jax = jax.jvp(
-        func_jax_with_axis, (test_regulararray_jax,), (test_regulararray_tangent_jax,)
-    )
-
-    value_vjp, vjp_func = jax.vjp(func_ak_with_axis, test_regulararray)
-    value_vjp_jax, vjp_func_jax = jax.vjp(func_jax_with_axis, test_regulararray_jax)
-
-    numpy.testing.assert_allclose(
-        ak.to_list(value_jvp), value_jvp_jax.tolist(), rtol=1e-9, atol=np.inf
-    )
-    numpy.testing.assert_allclose(
-        ak.to_list(value_vjp), value_vjp_jax.tolist(), rtol=1e-9, atol=np.inf
-    )
-    numpy.testing.assert_allclose(
-        ak.to_list(jvp_grad), jvp_grad_jax.tolist(), rtol=1e-9, atol=np.inf
-    )
-    numpy.testing.assert_allclose(
-        ak.to_list(vjp_func(value_vjp)[0]),
-        (vjp_func_jax(value_vjp_jax)[0]).tolist(),
-        rtol=1e-9,
-        atol=np.inf,
-    )
+        atol=1e-9,
+    )
+
+
+# TODO: Enable this when https://github.com/scikit-hep/awkward/issues/3541 is resolved.
```

Comment:
@ikrommyd - please, use skip instead of commenting out the test.
```suggestion
@pytest.mark.skip("sorting jax backend gives wrong results, see issue #3541 ")
```

Reply:
Maybe it should be marked `xfail` instead of just skipped to make it more visible

Reply:
Oh yea........I forgot `skip` and `xfail` exist haha

Reply:
Let's `xfail` it actually. I think it's best

Reply:
sounds good!

Reply:
Done!